src/main/java/core/basesyntax/service/action/BalanceAction.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.action;
+
+import core.basesyntax.model.Account;
+
+public class BalanceAction implements ActionHandler {
+    private Account account;
```

Comment:
The `Account` instance `account` is not initialized. This will lead to a `NullPointerException` when calling `setBananaBalance` or `setAppleBalance`. Consider initializing the `Account` instance or passing it through a constructor or setter method.